ch08.adoc

```diff
@@ -675,3 +675,130 @@ The data creator shall specify the floating-point arithmetic precision used duri
 Using the given computational precision in the interpolation computations is a necessary, but not sufficient, condition for the data user to be able to reconstitute the coordinates to an accuracy comparable to that intended by the data creator.
 For instance, a **`computational_precision**` value of **`"64"**` would specify that, using the same implementation and hardware as the creator of the compressed dataset, sufficient accuracy could not be reached when using a floating-point precision lower than 64-bit floating-point arithmetic in the interpolation computations required to reconstitute the coordinates.
 
+[[lossy-compression-via-quantization, Section 8.4, "Lossy Compression via Quantization"]]
+=== Lossy Compression via Quantization
+
+Geoscientific models and measurements generate false floating-point precision (scientifically meaningless data bits) that wastes storage space.
+False precision can mislead (by implying noise is signal) and is scientifically pointless.
+Quantization algorithms can eliminate false precision, usually by rounding the least significant bits of [<<IEEE_754>>] floating-point mantissas to zeros.
+(Quantization of integer types, although theoretically allowed, is not covered by this convention.)
+The quantized results are valid [<<IEEE_754>>] values---no special software or decoder is necessary to read them.
+Importantly, the quantized bits compress more efficiently than random bits.
+Thus quantization is sometimes referred to as a form of lossy compression although, strictly speaking, quantization only pre-conditions data for more efficient compression by a subsequent compressor.
+
+The CF conventions of this section define a metadata framework to record quantization properties alongside quantized floating-point data variables.
+The goals are twofold.
+First, to inform interested users how, and to what degree, the quantized data differ from the original unquantized data, which are not stored in the dataset and may no longer exist.
+Second, to provide the necessary provenance metadata for users to reproduce the data transformations on the same or other raw data.
+These conventions also allow users to better understand the precision that data producers expect from source models or measurements.
+Use of these conventions ensures that all quantized variables are clearly marked as such, and thus alerts users to cases where these guidelines have not been followed.
+
+These conventions must not be used with data variables of integer type, or any other kind of CF variable.
```

Comment:
At first I really liked this suggestion because it improve clarity and is more precise. However, it also may be too broad in a sense illustrated by the ancillary_variables in CF example 3.3. Would your suggested wording proscribe quantizing the data variable `q_error_limit` because it is referenced (as an `ancillary_variable`) by the `q` variable? If the dataset producer decides to quantize `q` (which seems reasonable), then it makes sense to me to allow quantizing other data variables like `q_error_limit` that are inherently known to the same precision as `q`. Thoughts?

Reply:
Good point. I don't think we want to prevent quantization of ancillary variables. We could instead say

> These conventions must only be used with variables of floating-point type that are not part of a domain defined by a domain variable (<\<domain-variables\>>) or data variable, i.e. are not acting as coordinates (of any type), cell measures, nor are terms in the definition of a parametric vertical coordinate (<\<parametric-vertical-coordinate\>>).

Reply:
I'm glad you agree that `ancillary_variables` is too broad of a class to rule out from potential quantization. Your suggested replacement sentence might be OK. However, it's hard for me to parse and I do not understand the logic of excluding variables that belong to a domain from being quantized. (NB: I just learned about domain variables today so I do not understand them that well). I agree that coordinate (of any type) variables that define a domain should never be quantized. Is that what you mean by variables that are "part of a domain"? To me "variables that are part of a domain" naturally includes data variables that occupy the domain, and I do not see the logic in proscribing quantization of these data variables. Please explain your reasoning or suggest a rewording that allows data variables in a domain to be quantized. Or are data variables and domain variables mutually exclusive so "data variables in a domain" is oxymoronic?

Reply:
@davidhassell When you have time please respond to above question.